WordPress/Classes/ViewRelated/Blaze/Webview/BlazeCampaignDetailsWebViewModel.swift

```diff
@@ -0,0 +1,115 @@
+import Foundation
+
+class BlazeCampaignDetailsWebViewModel: BlazeWebViewModel {
+
+    // MARK: Private Variables
+
+    private let source: BlazeSource
+    private let blog: Blog
+    private let campaignID: Int
+    private let view: BlazeWebView
```

Comment:
There may be a retain cycle here.

Ideally, ViewModel should not know about the View.

Reply:
> There may be a retain cycle here.
Good catch. Addressed in 1cf2cb1cc9fcf14645e84244ef6f34be950653af 👍 

> Ideally, ViewModel should not know about the View.
Good point, but that might require a big refactor that's out of the scope of this PR, IMHO.

Reply:
I agree. WebViews also don't really play well with ViewModels, so I think it's OK.